src/main/resources/mappings/anomaly-results.json

```diff
@@ -106,6 +106,60 @@
     },
     "model_id": {
       "type": "keyword"
+    },
+    "startOfAnomaly": {
```

Comment:
Can we use snake case like `start_of_anomaly` ?

Reply:
sure